dev/com.ibm.ws.jsf.2.2_fat/test-applications/JSF22SimpleHTML.war/src/com/ibm/ws/jsf22/fat/simple/bean/faces40/CompositeComponentTestBean.java

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright (c) 2015, 2019 IBM Corporation and others.
```

Comment:
Copyrights should be updated here (along with the other files in new packages), I believe?